lib/commands/init.ts

```diff
@@ -0,0 +1,20 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+
+export class InitCommand implements ICommand {
+	constructor(private $initService: IInitService) { }
+		
+	public allowedParameters: ICommandParameter[] = [];
+	public enableHooks = false;
+	
+	public execute(args: string[]): IFuture<void> {
+		return this.$initService.initialize();
+	}
+	
+	public canExecute(args: string[]): IFuture<boolean> {
```

Comment:
As you are always returning true, I believe it is better to use `
```TypeScript
public allowedParameters: ICommandParameter[] = [];
```
and remove canExecute. 
This way you'll get the benefit of commandsService canExecute logic, which will validate the number of parameters.
In case you keep this canExecute, the user will be able to write:
`tns init goshko peshko android test` and it will work just fine.